uint128_t/uint128_t_config.include

```diff
@@ -1,13 +1,13 @@
 #ifndef _UINT128_T_CONFIG_
   #define _UINT128_T_CONFIG_
   #if defined(_MSC_VER)
-    #if defined(_DLL)
-      #define _UINT128_T_EXPORT __declspec(dllexport)
-      #define _UINT128_T_IMPORT __declspec(dllimport)
-    #else
+//    #if defined(_DLL)
```

Comment:
the problem is that `_DLL` appears to be defined whenever a shared library is being built, even if `uint128_t` *itself* isn't built as a shared library. The python module is a shared library. I believe this macro is defined by cmake, since it doesn't seem to be defined by setup tools.

A fix to this should probably be upstreamed to `uint128_t` too, since this kind of configuration only allows all-or-nothing static or shared library builds.